ingestion/ingestion_server/ingestion_file_upload_rest_api.py

```diff
@@ -529,7 +540,6 @@ def manage_files():
     :param file_path:  Maps URL to files in templates directory.
     :return: Static file from directory along with data to display
     """
-    # TODO: Display all the files of the logged in owner only
     owner = current_user.get_id()
     if not current_user.get_id():    # To run ingestion as stand-alone
```

Comment:
you have already stored result of ```get_id()``` operation in ```owner```.

you also can replace this if statement with ternary expression: 
``` owner = current_user.get_id() if current_user.get_di() else 'Alice' ```